rest/config.go

```diff
@@ -1041,6 +1041,9 @@ func RunServer(config *ServerConfig) {
 
 	go sc.PostStartup()
 
+	base.Consolef(base.LevelInfo, base.KeyAll, "")
+	go config.Serve(":1234", CreateMetricHandler(sc))
```

Comment:
Port should be configurable, and presumably default to something other than 1234.